stf/src/saros/stf/Activator.java

```diff
@@ -60,4 +69,31 @@ public static Activator getDefault() {
   public static ImageDescriptor getImageDescriptor(String path) {
     return imageDescriptorFromPlugin(PLUGIN_ID, path);
   }
+
+  private void setupLoggers() {
```

Comment:
This now differs from the original logic. In the former logic only additional trace where enabled. Correct me if I am wrong but getStateLocation() returns a different directory than getStateLocation() inside the Saros Plugin itself.

Reply:
True, this logs the STF log calls into a separate dir/files.
Do you think it is more readable to have a single merged log file?

Reply:
I think it is stilled merged as the STF plugin uses the Log4J instance of the Core and not its own. However the intitalzation part will be written to the Saros/E log file and afterwards the log file is changed. Isn't that what this code is currently doing ?